docs/spec_checklist.md

```diff
@@ -0,0 +1,28 @@
+# StableHLO Specification Checklist
+
+In this document, we summarize the guidelines for reviewing changes to the
+specification. At the moment, these changes typically involve checking multiple
+things in multiple sources, so this document summarizes them all to simplify
+reviews:
+
+  1. Check if the section title matches the mnemonic in
+     [the ODS](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/StablehloOps.td).
+  1. Check if the "Semantics" section matches XLA's
```

Comment:
numberings need to be fixed.

Reply:
I don't think they do - that's the markdown notation for numbered lists. If you click "View file", these ones are replaced with consecutive numbers.